crates/vm/levm/src/errors.rs

```diff
@@ -78,9 +86,10 @@ pub enum VMError {
     ConsumedGasOverflow,
     #[error("Max Gas Limit Exceeded")]
     MaxGasLimitExceeded,
-    // Internal
-    #[error("Internal error: {0}")]
-    Internal(#[from] InternalError),
+    #[error("Arithmetic operation overflowed in gas calculation")]
+    ArithmeticOperationOverflow,
```

Comment:
Isn't this a `GasCostOverflow`?

Reply:
Replaced it with GasCostOverflow: [Remove ArithmeticOperationOverflow and replace it](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1138/commits/adef3537f1b2243b23b7a2c5341dd93dfd7bd432)